README.md

```diff
@@ -1,6 +1,7 @@
 # Docker ELK stack
 
 [![Join the chat at https://gitter.im/deviantony/docker-elk](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/deviantony/docker-elk?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Elastic Stack version](https://img.shields.io/badge/ELK-5.6.2-blue.svg?style=flat)](https://github.com/deviantony/docker-elk/issues/175)
+[![Build Status](https://api.travis-ci.org/ivanrome/docker-elk.svg?branch=master)](https://travis-ci.org/ivanrome/docker-elk)
```

Comment:
Consider changing `ivanrome` to `deviantony` in the URL before we merge :)

Reply:
OK :smile: